### PR TITLE
[IoTHub] Breaking Changes for manual-failover command

### PR DESCRIPTION
**Pull request by anusapan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by fengzhou-msft** [comment]
@Juliehzl please help with this PR.

File: src/azure-cli/azure/cli/command_modules/iot/_help.py [hunk 1]
```diff
@@ -485,9 +485,9 @@
 type: command
 short-summary: Initiate a manual failover for the IoT Hub to the geo-paired disaster recovery region.
 examples:
-  - name: This fails over “myhub” from East US to West US.
+  - name: Initiating failover “myhub” from primary to secondary region.
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/iot/_help.py]: Base on CLI convention, please use `Initiate`

**Comment by Juliehzl** [comment]
There is no usage request for --failover-region for the past 30 days. Breaking change is acceptable.

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -50,6 +50,7 @@ Release History
 * Fix #2531: Add convenience arguments for hub update.
 * Fix #8323: Add missing parameters to create storage custom endpoint.
 * Fix regression bug: Reverting the changes which overrides the default storage endpoint.
+* [BREAKING CHANGE] Removed --failover-region parameter from manual-failover. Now it will failover to assigned geo-paired secondary region.
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: Please rebase dev branch and place your release not in the right position for next release.

File: src/azure-cli/azure/cli/command_modules/iot/_help.py (hunk 1 above)
Comment (anusapan) [reply src/azure-cli/azure/cli/command_modules/iot/_help.py]: fixed

File: src/azure-cli/HISTORY.rst (hunk 2 above)
Comment (anusapan) [reply src/azure-cli/HISTORY.rst]: done

**Pull request merged**